index-fr.html

```diff
@@ -108,14 +107,14 @@ <h2>Diffusez le message, pas le virus</h2>
 
     <p>
       Il est important de se souvenir qu'aucun masque n'est parfait.
-      Même les masques FFP2 ou N95 recommandés pour le personnel de santé ne garantissent de bloquer que <a href="https://www.youtube.com/watch?v=eAdanPfQdCA">95% des particules les plus fines</a> (et cela seulement si vous les <a href="https://www.n95decon.org/publications">portez correctement</a>).
+      Même les masques FFP2 ou N95 recommandés pour le personnel de santé ne garantissent pas de bloquer que <a href="https://www.youtube.com/watch?v=eAdanPfQdCA">95% des particules les plus fines</a> (et cela seulement si vous les <a href="https://www.n95decon.org/publications">portez correctement</a>).
```

Comment:
"pas" is not correct here. We want to say that the mask guarantee a limited level of protection, and "pas" would introduce a double negative. I agree that the sentence is a bit uncomfortable to read though, so I'm open to a full rewrite if you can find something better.